### PR TITLE
HPCC-17224 Fix INIT_PATH in configure target for development builds

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@mayx 
`make configure` should now properly setup .ssh keys and lay down INIT_PATH with symlinks to the freshly built dafilesrv/hpcc-init files.  Can you test this out and see if it fixes the issue you were having with development builds?

**Comment by richardkchapman** [comment]
A few checkboxes missing on the checklist...

**Comment by Michael-Gardner** [comment]
@mayx I added in the runtime directories to the configure target.  Also you won't have to call the configure target anymore explicitly.  It gets added to the ALL target.

**Comment by Michael-Gardner** [comment]
@xwang2713 Please review this when you get a chance.

**Review by xwang2713 — COMMENTED** [review]
Looks good

**Comment by Michael-Gardner** [comment]
@richardkchapman ready for merge

File: initfiles/etc/DIR_NAME/CMakeLists.txt [hunk 1]
```diff
@@ -2,9 +2,7 @@
 #    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
 #
 #    All rights reserved. This program is free software: you can redistribute it
-#    and/or modify
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
+#    and/or modify #    you may not use this file except in compliance with the License.  #    You may obtain a copy of the License at
```
Comment (richardkchapman) [inline initfiles/etc/DIR_NAME/CMakeLists.txt]: This change looks incidental (and wrong)

**Comment by richardkchapman** [comment]
@Michael-Gardner One minor comment

**Comment by Michael-Gardner** [comment]
@richardkchapman Fixed

**Pull request merged**